src/index.html

```diff
@@ -11,7 +11,226 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-div-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="personalinfo">
+        <legend>
+          Personal information
+        </legend>
+        <div class="first">
+          <label for="surname">
```

Comment:
https://github.com/mate-academy/layout_html-form/pull/189#discussion_r1087765248
you don't need both